scapy/layers/dhcp6.py

```diff
@@ -1026,6 +1029,16 @@ class DHCP6OptClientLinkLayerAddr(_DHCP6OptGuessPayload):  # RFC6939
                    _LLAddrField("clladdr", ETHER_ANY)]
 
 
+class DHCP6OptMudUrl(_DHCP6OptGuessPayload):  # RFC8520
+    name = "DHCP6 Option - MUD URL"
+    fields_desc = [ShortEnumField("optcode", 112, dhcp6opts),
+                   FieldLenField("optlen", None, length_of="mudString"),
+                   StrLenField("mudString", "",
```

Comment:
Could you use mudstring in lowercase instead?

Reply:
Sure, I added two fixup commits adressing this :)

Reply:
Hmm, the code coverage percentage seems to have decreased by a significant amount.  Do you see a way for me to fix this?